README.md

```diff
@@ -55,11 +55,15 @@ Changes to the CDM may be proposed by CDM Working Groups, individual corporate c
 
 **2.2.2 Appointment of CDM Steering Working Group Maintainers:**
 
-* At the launch of the project, up to two initial Maintainers will be nominated by ICMA, ISDA, and ISLA (collectively, the “trade associations”).
-* Additional CDM Steering Group  Maintainers may be proposed by Participants. Proposed maintainers will be approved via consensus of the Participants and with agreement of existing Maintainers, and should meet the following criteria:
+* Maintainers can be of one of two types: Organisational Representative (OrgRep) or Private individual. Private individuals are maintainers in their own right, acting on their own volition and not representing another legal entity. OrgReps are appointed by their organisation, acting on behalf and for that organisation. In this case it is the organisation which will be subject to any vote on maintainer membership. Organisations are recommended to appoint at least two representatives for coverage purposes. Those representatives can be changed at any time, without any voting process by other maintainers or community Participants, by the organisation in order to cater for staff changes. 
+A list of Maintainers along with their type will be kept visible within the governance documentation provided on the repo and mini-site.
+
+* Additional CDM Maintainers may be proposed by Participants. Proposed new Private individuals or new organisations will be approved via consensus of the Participants and with agreement of existing Maintainers, and should meet the following criteria:
     * Proven experience in data modelling and/or software development in financial markets.
     * In-depth understanding and proven track record of contribution to the CDM, as well as other data standards (such as ISO) and messaging protocols (such as FIX, FpML or Swift).
 
+*Private individuals and Organisations may choose to rescind their maintainership at will but must notify the other maintainers. OrgReps may be replaced by their organisations at any time. Any entity who has chosen to rescind their maintainership, but then at a later time would like to re-join the maintainer team, is subject to the voting process outlined above. Additionally any OrgRep who has been replaced by their organisation, but wishes to remain a maintainer as a Private Individual will also need to go through the voting process)
```

Comment:
There is a close bracket right at the end of this line which does not have an opening bracket.